pypeit/reduce.py

```diff
@@ -1332,6 +1370,80 @@ def global_skysub(self, skymask=None, update_crmask=True, trim_edg=(0,0),
             sobjs_show = None if show_objs else self.sobjs_obj
             # Global skysub is the first step in a new extraction so clear the channels here
             self.show('global', slits=True, sobjs=sobjs_show, clear=False)
+        return self.global_sky
+
+    def global_skysub(self, skymask=None, update_crmask=True, trim_edg=(0,0),
+                      show_fit=False, show=False, show_objs=False):
+        """
+        Perform global sky subtraction. This IFU-specific routine ensures that the
+        edges of the slits are not trimmed, and performs a spatial and spectral
+        correction using the sky spectrum, if requested. See Reduce.global_skysub()
+        for parameter definitions.
+        """
+        # Generate a global sky sub for all slits separately
+        global_sky_sep = Reduce.global_skysub(self, skymask=skymask, update_crmask=update_crmask, trim_edg=trim_edg,
```

Comment:
We have generally done a `super` call, although I admit I don't know the difference.